pkg/monitoring/rules/recordingrules/vmi.go

```diff
@@ -10,9 +10,10 @@ var vmiRecordingRules = []operatorrules.RecordingRule{
 	{
 		MetricsOpts: operatormetrics.MetricOpts{
 			Name: "cnv:vmi_status_running:count",
-			Help: "The total number of running VMIs by status",
+			Help: "The total number of running VMIs by phase, labeled with node, instance type, preference and guest OS information",
```

Comment:
@assafad I think it should be:
The total number of running VMIs, labeled with node, instance type, preference and guest OS information

Reply:
Agree, done.